src/cms/api/preview.php

```diff
@@ -1,22 +1,18 @@
 <?php
 
```

Comment:
this file saves the current edited page (but WITHOUT the cms included) to preview.html ... which is then shown inside an iframe by the CMS when u click "Preview" in the main menu